include/swift/SIL/SILCloner.h

```diff
@@ -1252,15 +1297,28 @@ void
 SILCloner<ImplClass>::visitWitnessMethodInst(WitnessMethodInst *Inst) {
   auto conformance =
     getOpConformance(Inst->getLookupType(), Inst->getConformance());
+  auto lookupType = Inst->getLookupType();
+  auto newLookupType = getOpASTType(lookupType);
+  SILValue OpenedExistential;
+  if (Inst->hasOperand())
+    OpenedExistential = getOpValue(Inst->getOperand());
```

Comment:
Actually, with all the introduced machinery in place, we can now handle the WitnessMethodInst as any regular instruction that may have opened archetype operands. There is no need for any special handling.